rcl/src/rcl/arguments.c

```diff
@@ -51,12 +53,22 @@ _rcl_parse_remap_rule(
   rcl_allocator_t allocator,
   rcl_remap_t * output_rule);
 
+/// Parse an argument that may or may not be a parameter file rule.
+/// The syntax of the file name is not validated.
+/// \param[in] arg the argument to parse
+/// \param[in] allocator an allocator to use
+/// \param[in,out] param_file string that could be a parameter file name
+/// \return RCL_RET_OK if the rule was parsed correctly, or
+/// \return RCL_RET_INVALID_PARAM_RULE if the argument is not a valid rule, or
+/// \return RCL_RET_BAD_ALLOC if an allocation failed, or
+/// \return RLC_RET_ERROR if an unspecified error occurred.
+/// \internal
```

Comment:
nitpick: since this is not parsed by doxygen, I'd recommend using normal comments, e.g. `//` or `/* text */`.

Reply:
while I was mostly just copying the existing formatting, I had imagined from the /internal tag that we were using doxygen formatting to optionally support [building internal docs](https://www.stack.nl/~dimitri/doxygen/manual/config.html#cfg_internal_docs). But even without that could it still be worthwhile keeping the doxygen formatting in case one day we move them to the header? Or do you find it easier to read as a standard comment?

Reply:
Well if you want to use doxygen style that’s fine, though we should turn on the internal docs feature then. But I’m sure this is something I mentioned on the other prs, but this doesn’t match the doxygen style for most of rcl. So if you’re going to touch it, I’d recommend the style described in our style guide under comments and doc comments: https://github.com/ros2/ros2/wiki/Developer-Guide#c-1

Reply:
yep, happy to change the file to use /** style. and will look at turning on the internal docs generation